apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -86,23 +85,24 @@ const getAvailableTabs = permissions => {
   return tabs;
 };
 
-const getEnrollSucessWorkshopName = () => {
+const getEnrollSucessWorkshopTitle = () => {
   // If sent here from successfully enrolling in a workshop, log WORKSHOP_ENROLLMENT_COMPLETED_EVENT.
-  const urlParams = queryParams();
-  if (urlParams && Object.keys(urlParams).includes('wsCourse')) {
-    const workshopCourseName = urlParams['wsCourse'];
+  const workshopCourse = sessionStorage.getItem('workshopCourse', null);
+  if (!workshopCourse) {
+    return '';
+  } else {
+    const workshopName = sessionStorage.getItem('workshopName', null);
 
     analyticsReporter.sendEvent(EVENTS.WORKSHOP_ENROLLMENT_COMPLETED_EVENT, {
-      'regional partner': urlParams['rpName'],
-      'workshop course': workshopCourseName,
-      'workshop subject': urlParams['wsSubject'],
+      'regional partner': sessionStorage.getItem('rpName', null),
```

Comment:
I switched to using `sessionStorage` since several parameters were being passed in and the workshop name could be up to 255 characters long so I felt storing it made more sense than appending as part of the URL.